app/src/main/res/layout/fragment_movie_detail.xml

```diff
@@ -0,0 +1,108 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".fragments.MovieDetailFragment"
+    android:background="@color/background_teal"
+    >
+
+    <TextView
```

Comment:
Looking at this xml, I assume you used the designer to arrange your elements, but this has led to some weird things and numbers which to be honest I don't like. 
Avoid such specifikcs as 336, 49, 143 dps.. except if your view is like really really custom. and you can not avoid it.
Also the biases are ... "0.493".
I don't think all this is necessary, your layout is becoming too complex and concrete and every new elemnt can break the whole thing.